web-client/package-lock.json

```diff
@@ -2554,7 +2653,7 @@
     },
     "brfs": {
       "version": "1.6.1",
-      "resolved": "https://registry.npmjs.org/brfs/-/brfs-1.6.1.tgz",
+      "resolved": "http://registry.npmjs.org/brfs/-/brfs-1.6.1.tgz",
```

Comment:
This behavior has been observed elsewhere, and it's a product of using the "npm" tools.
https://npm.community/t/npm-install-downgrading-resolved-packages-from-https-to-http-registry-in-package-lock-json/1818/3
There is an opportunity to fix/replace these to ensure the package-lock dependencies always use "https" if you wish.

Reply:
Yeah, I'm just not comfortable retrieving packages via HTTP, especially if there's an option to use TLS — that's not going to survive a security review on the way to production. I'd like to see us ensure the use of HTTPS links. If that's plausible to do in the context of this code review, great, but if that's a heavy lift, it can wait for the end of sprint 5.